kernel/cpu_vectors_aarch64.S

```diff
@@ -0,0 +1,121 @@
+/* Linkage for ARM */
+#define __ALIGN .align 2
+#define __ALIGN_STR ".align 2"
+
+#define ALIGN __ALIGN
+#define ALIGN_STR __ALIGN_STR
+
+#define ENTRY(name)                             \
+  .globl name;                                  \
+  ALIGN;                                        \
+  name:
+
+#define GLOBAL(name)                            \
+  .globl name;                                  \
+  name:
+
+#define END(name) \
+  .size name, .-name
+
+#define ENDPROC(name) \
+  .type name, %function; \
+  END(name)
+
+/*
+ * Exception trap types
+ *-----------------
+ */
+#define ET_SYNC    0
+#define ET_IRQ     1
+#define ET_FIQ     2
+#define ET_ERROR   3
+
+/*
+ * Invalid exception handlers
```

Comment:
That's because, we think all exceptions we get in guest are invalid. For example, If we got a page fault exception in guest. Because we have allocated memory for all pages. This page fault exception is abnormal, so it's invalid, we have to panic the guest.